Objects/EventItem/Chicken.gd

```diff
@@ -2,3 +2,11 @@ extends "res://Objects/EventItem/EventItem.gd"
 
 func _ready():
 	set_start_position(600, 293)
+
+func _process(delta):
+	position.x += move_speed * delta
+
+func _on_Item_pressed():
```

Comment:
This code is the same as in `EventItem.gd` so we don't need to override this function. I recommend to remove it.